pymodbus/client/mixin.py

```diff
@@ -695,18 +695,19 @@ class DATATYPE(Enum):
 
     @classmethod
     def convert_from_registers(
-        cls, registers: list[int], data_type: DATATYPE
+        cls, registers: list[int], data_type: DATATYPE, endian: Literal["big", "little"] = "big"
     ) -> int | float | str | list[bool]:
         """Convert registers to int/float/str.
 
         :param registers: list of registers received from e.g. read_holding_registers()
         :param data_type: data type to convert to
+        :param endian: endianness of the byte order
```

Comment:
This is wrong, byte order is always big.

What is not defined by the standard is the word order.